README.md

```diff
@@ -9,13 +9,13 @@
 
 ## Description
 
-Patch-level verification for [bundler].
+The best tool to find leaky gems in your dependencies. Make sure your memory leaks
```

Comment:
Should this be "Make sure memory leaks"  instead of " Make sure your memory leaks" ?

Reply:
@bronzdoc That sounds better. I just updated this PR.